frontend/test/views/AnnotationView.spec.js

```diff
@@ -116,6 +117,12 @@ describe('AnnotationView', () => {
     expect(textDatasets.length).to.equal(9);
   });
 
+  // Enable this test when tag datasets are added to the configuration
```

Comment:
https://vitest.dev/api/#test-skip This comment and test were helpful for the PR.  You can do something similar that will include it in the test suite without leaving it commented out.  All unit test frameworks have ways to configure a 'skipped' unit test. This would be an applicable case.  The comment above can be left in to remind us to remove the skip once we are permanently using the component.  update this to use .skip instead.